Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.py

```
@@ -297,17 +299,45 @@ def http_request(url_suffix, method='POST', data={}, err_operation=None):
     return parse_response(res, err_operation)
 
 
-def validate_sort_and_order_and_artifact(sort, order, artifact_source):
+def validate_sort_and_order_and_artifact(sort: Optional[str], order: Optional[str], artifact_source: Optional[str])\
+        -> Optional[str]:
+    """
+    Function that validates the arguments combination.
+    sort and order arguments have come together.
+    Sort and order cant appear with artifact.
+    Args:
+        sort: variable to sort by.
+        order: the order which the results is ordered by.
+        artifact_source: true if artifacts are needed and false otherwise.
+    Returns:
+        true if arguments are valid for the request, false otherwise.
+    """
     if artifact_source == 'true' and sort:
-        return_error('artifact and sort are not supported for scan and scroll together, Please Remove one.')
+        return_error('Please remove or disable one of sort or artifact, As they are not supported in the api together.')
     elif sort and not order:
         return_error('Please specify the order of sorting (Ascending or Descending).')
```

let's raise an exception as there is try/catch for the functions